src/models/CNNRGB/CNNRGB-standing_laying/q4-rgb-standing-lying/src/data_preparation/data_preparation.py

```diff
@@ -0,0 +1,97 @@
+# Mount the container with the compute instance
+
+import pandas as pd
+import random
+from shutil import copyfile
+from PIL import Image
+import os
+
+#  Parent directory
+PARENT_DIRECTORY = '/mnt/preprocess/'
+BASE_DIRECTORY_PATH = os.path.join(PARENT_DIRECTORY, 'standing_laying_v1.0/')
+if not os.path.exists(BASE_DIRECTORY_PATH):
+    os.mkdir(BASE_DIRECTORY_PATH)
+
+#  Train directory
+TRAIN_DIRECTORY_PATH = os.path.join(BASE_DIRECTORY_PATH, 'train/')
+if not os.path.exists(TRAIN_DIRECTORY_PATH):
+    os.mkdir(TRAIN_DIRECTORY_PATH)
+
+#  Standing directory
+STANDING_PATH = os.path.join(TRAIN_DIRECTORY_PATH, 'standing/')
+if not os.path.exists(STANDING_PATH):
+    os.mkdir(STANDING_PATH)
+
+#  laying directory
+LAYINGING_PATH = os.path.join(TRAIN_DIRECTORY_PATH, 'laying/')
+if not os.path.exists(LAYINGING_PATH):
+    os.mkdir(LAYINGING_PATH)
+
+
+#  Read CSV from the ML Database
+#  To generate CSV use 'select_artifacts_with_targets' view with change in Dataformat(jpg)
+
+artifacts = pd.read_csv('training_data.csv')
+print("Shape of artifacts in ML database" + artifacts.shape)
+
+#  To get only good artifacts
+artifacts = artifacts[artifacts['tag'] == 'good']
+print("Shape of artifacts tagged good" + artifacts.shape)
+
+artifacts.head()
+
+del artifacts['tag']
+del artifacts['scan_group']
+del artifacts['id']
+
+#  label the Data between standing and laying
+#  1 is standing
+#  0 is laying
+artifacts['label'] = artifacts.apply(lambda x: 1 if int(int(x['key']) / 100) == 1 else 0, axis=1)
+
+#  Print the value count in artifacts level
+print(artifacts['label'].value_counts())
+
+#################################################################
+#  This function copy the artfacts from one folder to another  ##
+#  and Rotatting the laying child by 90 Anti-clockwise and     ##
+#  Standing child by 270 Anti-clockwise                        ##
+
+
+def find_artifacts_by_label(label, start_point, end_point, artifacts):
```

Comment:
This comment looks like it doesn't belong. It wants to be a docstring, right?

```suggestion
def find_artifacts_by_label(label, start_point, end_point, artifacts):
   """Copy the artifacts from one folder to another
        and rotating the laying child by 90 anti-clockwise and
        standing child by 270 anti-clockwise
   """
```